admin/views/tool-file-editor.php

```diff
@@ -32,7 +32,9 @@
 	do_robots();
 	$robots_content = ob_get_clean();
 
+	// phpcs:ignore WordPress.WP.AlternativeFunctions.file_system_read_fopen We think WP_Filesystem is not always predictable.
```

Comment:
```suggestion
	// phpcs:ignore WordPress.WP.AlternativeFunctions.file_system_read_fopen -- We think WP_Filesystem is not always predictable.
```
(here and everywhere else)

Re: the ignoring - You are ignoring the reason why the WP file system exists, i.e. that depending on hosting setup etc, a site _may_ not have direct access to the file system and would need to use FTP or Curl if `fopen` is disabled etc.

If the WP Filesystem is unpredictable, that should probably be fixed instead of not using it. Is there a Trac ticket open about this ?

In the mean time, as a temporary solution, I can understand the choice for ignoring the issues, so it's as if I'm rejecting this PR.

On a practical note: for a function like this one, you the ignores less noisy, by using `phpcs:disable ....` at the start of the function/above the first filesystem function call in the function and then adding `phpcs:enable` at the end of the function/after the last filesystem function call.

For an example, see: https://github.com/Yoast/wordpress-seo-premium/pull/2912/commits/d09a9822d4377022f1f5cb997db7fc9f6bd52481